src/components/NewClusterModal.js

```diff
@@ -386,7 +400,29 @@ export const NewClusterModal = withModalDrawer({ width: 675 })(class NewClusterM
           ])
         ])
       })],
-      [Utils.DEFAULT, () => ({
+      [isDeleteView === true, () => ({
+        onPrevious: () => this.setState({ isDeleteView: false }),
+        contents: h(Fragment, [
+          p({ style: { margin: '0px' } }, [
+            'Deleting your runtime configuration will stop all running notebooks and associated costs. '
```

Comment:
TOL: I know this language is mid-transition, but calling it 'configuration' seems like a miscommunication to me. What we're doing is deleting __the runtime__, which is a fairly heavy and destructive operation, and we should make sure we're representing that accurately. (I'm thinking here about more knowledgable users for whom 'configuration' might sound like something lighter, e.g. a yaml file, rather than a whole VM.)